distributed/bokeh/worker.py

```diff
@@ -678,6 +678,19 @@ def __init__(self, worker, io_loop=None, prefix='', **kwargs):
         self.loop = io_loop or worker.loop
         self.server = None
 
+    @property
+    def extra(self):
+        return merge({'prefix': self.prefix}, template_variables)
+
     @property
     def my_server(self):
         return self.worker
+
+    def listen(self, *args, **kwargs):
+        super(BokehWorker, self).listen(*args, **kwargs)
+
+        from .worker_html import routes
+        handlers = [(self.prefix + '/' + url, cls, {'server': self.my_server, 'extra': self.extra})
+                    for url, cls in routes]
+
+        self.server._tornado.add_handlers(r'.*', handlers)
```

Comment:
Thanks for doing this.  It's nice to have the infrastructure setup on the worker to add new routes in the future